### PR TITLE
Check stars_kws and gals_kws for valid entries

### PR DESCRIPTION
**Pull request by esheldon**

### CODE REVIEW

File: descwl_shear_sims/simple_sim.py [hunk 1]
```diff
@@ -576,12 +587,20 @@ def _setup_stars(
...
+                self.stars_kws,
+                ('density', 'mag', 'min_mag'),
+                'stars_kws',
+            )
+            if ('density' in self.stars_kws and
+                    isinstance(self.stars_kws['density'], dict)):
+                check_keys(
+                    self.stars_kws['density'],
+                    ('min_density', 'max_density'),
+                    'stars_kws["density"]',
```
Comment (beckermr) [inline descwl_shear_sims/simple_sim.py:600]: Can we pull the list of valid keys from the defaults? This will make maintenance easier.

Comment (beckermr) [reply descwl_shear_sims/simple_sim.py:600]: Or at least we could store them at the top of the file.

Comment (esheldon) [reply descwl_shear_sims/simple_sim.py:600]: they are not in the defaults

Comment (beckermr) [reply descwl_shear_sims/simple_sim.py:600]: top of the file would be good too.

**Pull request merged**